pkg/volume/plugins.go

```diff
@@ -412,6 +412,9 @@ type VolumeHost interface {
 	// Returns host IP or nil in the case of error.
 	GetHostIP() (net.IP, error)
 
+	// Returns HostUtils Interface
+	GetHostUtil() mount.HostUtils
```

Comment:
We're trying to start separating out VolumeHost into controller-specific interfaces.  Since this is only used by kubelet, can you add this to KubeletVolumeHost?